include/items.php

```diff
@@ -150,8 +150,9 @@ function add_page_info_data($data) {
 		is_string($data["text"]) AND (sizeof($data["images"]) > 0))
 		$data["type"] = "link";
 
-	if ((($data["type"] != "link") AND ($data["type"] != "video") AND ($data["type"] != "photo")) OR ($data["title"] == $url))
+	if ((($data["type"] != "link") AND ($data["type"] != "video") AND ($data["type"] != "photo")) OR ($data["title"] == $data["url"])) {
 		return("");
```

Comment:
Standards:  language construct like `return`, `include`, `require` and `echo` should not have parenthesis.